package.json

```diff
@@ -22,7 +22,9 @@
     "fast-safe-stringify": "^2.0.7",
     "lodash.isempty": "^4.4.0",
     "lodash.iserror": "^3.1.1",
-    "lodash.isundefined": "^3.0.1",
-    "p-queue": "^8.1.0"
```

Comment:
not a must but maybe since we remove this we can return the package to be like it used to (common js) and not module

Reply:
Talked about it with @yaronya and since we already made the change it would be redundant to bring it back to common js but I will remove the redundant package